### ISSUE TITLE
LuCI offers WPS in 802.11s encryption

### ISSUE THREAD

**Issue by castillofrancodamian** [openwrt/luci#5830]
With the default package `wpad-basic-wolfssl` it doesn't offer it, but with `wpad-mesh-wolfssl` it does. Even in `Access Point` mode too.
I don't have the `hostapd-utils` package installed.

`wpad-basic-wolfssl` and `802.11s` interface.

![Screenshot_20220602_164546](https://user-images.githubusercontent.com/47670442/171729512-917f32ad-d742-4021-a83f-3f48a12d7a90.png)

`wpad-mesh-wolfssl` and `802.11s` interface.

![Screenshot_20220602_170016](https://user-images.githubusercontent.com/47670442/171729565-464842e6-5f36-4167-8ba7-abc5741ea617.png)

`wpad-mesh-wolfssl` and `Access Point` interface.

![Screenshot_20220602_170041](https://user-images.githubusercontent.com/47670442/171729697-71ef0114-bb9a-400f-8018-7029339edb00.png)

**Comment by jow-** [comment]
And what is the problem exactly?

**Comment by castillofrancodamian** [comment]
I don't see that it's a problem, but I find it curious that it offers something that can't be used.

**Comment by jow-** [comment]
So you mean to say that the WPS option should be hidden when 11s mode is selected because it cannot be used in 11s mode? Please describe in clear and simple words what the problem is, I don't understand what you want me to change or fix from the ticket description.

**Comment by castillofrancodamian** [comment]
Yes. I think it has to be hidden because it can't be used in any way that I know of. The simple solution is to not see that option. I can't tell if it's a problem, but I'm curious. It is like in the encryption sector that offers all types of encryption and the only usable one is WPA3-SAE.
